### ISSUE TITLE
Timer needs more than * microseconds

### ISSUE THREAD

**Issue by Fedeorlandau** [TeaSpeak/TeaSpeak#363]
**Describe the bug**
Hi All, I'm having the following error in my logs:
```[2018-08-25 23:30:09] [ERROR] Timer InstanceHandler::tickInstance -> statistics tick needs more than 5134 microseconds. Max allowed was 5000 microseconds.
[2018-08-25 23:30:09] [DEBUG]     2 | [Property] Changed property in db key: client_month_online_time value: 960
[2018-08-25 23:30:09] [DEBUG]     2 | [Property] Changed property in db key: client_total_online_time value: 960
[2018-08-25 23:30:09] [ERROR] Timer ConnectedClient::tick needs more than 2292 microseconds. Max allowed was 2000 microseconds.
[2018-08-25 23:30:09] [ERROR]     2 | Ticking of client 190.193.65.18:55891 (Poseidon) needs more that 2500 microseconds! (3872 microseconds)
[2018-08-25 23:30:11] [ERROR] Timer VoiceClient::tick needs more than 7588 microseconds. Max allowed was 3000 microseconds.
[2018-08-25 23:30:11] [ERROR]     1 | Ticking of client 191.119.111.242:59657 (Zeus) needs more that 2500 microseconds! (9257 microseconds)
[2018-08-25 23:30:13] [ERROR] Timer VoiceClient::tick needs more than 6478 microseconds. Max allowed was 3000 microseconds.
[2018-08-25 23:30:13] [ERROR]     2 | Ticking of client 190.193.65.18:55891 (Poseidon) needs more that 2500 microseconds! (6839 microseconds)
[2018-08-25 23:30:14] [ERROR] Timer VoiceClient::tick needs more than 6525 microseconds. Max allowed was 3000 microseconds.
[2018-08-25 23:30:14] [ERROR]     1 | Ticking of client 191.119.111.242:59657 (Zeus) needs more that 2500 microseconds! (8694 microseconds)
[2018-08-25 23:30:14] [ERROR]     2 | Ticking of client 190.193.65.18:55891 (Poseidon) needs more that 2500 microseconds! (10447 microseconds)
[2018-08-25 23:30:16] [ERROR]     1 | Ticking of client 179.7.225.243:50474 (Ebrians / Error) needs more that 2500 microseconds! (2784 microseconds)
[2018-08-25 23:30:16] [ERROR]     1 | Ticking of client 181.95.219.242:53661 (Ramsay) needs more that 2500 microseconds! (7572 microseconds)
[2018-08-25 23:30:21] [ERROR] Timer InstanceHandler::tickInstance -> statistics tick needs more than 15 milliseconds. Max allowed was 5 milliseconds.
[2018-08-25 23:30:21] [ERROR] Timer ConnectedClient::tick needs more than 3286 microseconds. Max allowed was 2000 microseconds.
[2018-08-25 23:30:21] [ERROR]     2 | Ticking of client 190.193.65.18:55891 (Poseidon) needs more that 2500 microseconds! (7494 microseconds)
[2018-08-25 23:30:22] [ERROR]     4 | Found varianzes within the server tick! (Supposed: 500ms Hold: 518ms)
[2018-08-25 23:30:22] [ERROR]     4 | Found varianzes within the server tick! (Supposed: 500ms Hold: 478ms)
[2018-08-25 23:30:23] [ERROR] Timer VoiceClient::tick needs more than 3134 microseconds. Max allowed was 3000 microseconds.
[2018-08-25 23:30:23] [ERROR]     2 | Ticking of client 190.193.65.18:55891 (Poseidon) needs more that 2500 microseconds! (5393 microseconds)
[2018-08-25 23:30:26] [ERROR] Timer InstanceHandler::tickInstance -> statistics tick needs more than 6433 microseconds. Max allowed was 5000 microseconds.
[2018-08-25 23:30:28] [ERROR]     2 | Ticking of client 190.193.65.18:55891 (Poseidon) needs more that 2500 microseconds! (4521 microseconds)
```

Also, I guess this is causing some reconnects and if you want to connect to the server you have to try twice before entering.

Any clues on this?

**Comment by AndreyRigid** [comment]
Yesterday there was the same problem. Even sinned on VPS, so the launch of the same server locally did not cause these problems. But the transfer to another server did not help. A new server with an empty database works almost without brakes, but there are similar errors, but with less critical numbers. And the server that has worked for 2 weeks does not go, and if it works, then the transition from channel to channel lasts 10-15 seconds. Clients enter for 10 seconds after which they are thrown out and they are not sent data about the channels.

**Comment by WolverinDEV** [comment]
Well in general this messages arent a problem.
They are just "notifications" that something is slower than expected, may cause by a lot of operations or some similar stuff.

@AndreyRigid 
Well can I ask you what a "FULL" database means for you?

**Comment by AndreyRigid** [comment]
@WolverinDEV Это значит со всеми необходимыми каналами, группами и их разрешениями. 
И я уверен, что зависания, не позволяющие заходить на сервер, это всё-таки проблема. Да и когда всё таки удаётся зайти на сервер и переход с канала на канал занимает 10 секунд - это тоже проблема. Тем более когда на ранней версии сервера с той же базой нет таких проблем.

**Comment by WolverinDEV** [comment]
Did you tried the newest version because 2 builds ago i forgot to reactivate a caching process which caused a lot of cpu usage.

**Comment by AndreyRigid** [comment]
I installed the last version last night - normal work was with a small number of users and in prime everything hung (about 40 users ), this morning I had to install the old one - there's no problem with it. In general, the problems began after the update to the latest version on August 24.
2 builds ago lags were immediately upon entering, but now it's not easier.
At all equal - with the old version there are no problems.
